proposed/2021/VulnerabilityInfoInPMUI.md

```diff
@@ -0,0 +1,52 @@
+# Expose package vulnerability information in the Package Manager UI - Technical Spec
+
+* Status: **In Review**
+* Author(s): [Jean-Pierre Briedé](https://github.com/jebriede)
+* Issue: [10981](https://github.com/NuGet/Home/issues/10981) Expose package vulnerabilities information in the PMUI
+* Type: Feature
+
+## Problem Background
+
+Developers that install NuGet packages through Visual Studio may be installing packages that have known vulnerabilities or they may have a project that is depending on a vulnerable version of a NuGet package. There is currently no information in the Package Manager UI that indicates that a package version is vulnerable so developers may continue to use a vulnerable package in their project. Users should be able to see a warning about a package version they are browsing or one they have installed so they can be informed of the vulnerability and hopefully update to a different version of the package that is not vulnerable or use a different package.
+
+## Who are the customers
+
+* All users that install a NuGet packages using the Package Manager in Visual Studio.
+* All users that manage their installed NuGet packages using the Package Manager in Visual Studio.
+
+## Requirements
+
+* A warning icon will be shown in the Installed tab and in the package list if the project has any vulnerable packages installed. These icons will be the same ones that are used when deprecated packages are installed. In the case where a package version is both vulnerable and deprecated, only one icon will still be shown, but the tooltip on the icon will reflect the vulnerability and deprecation.
```

Comment:
Do we have any plan to display these vulnerabilities differently based on the severity of it?

https://github.com/NuGet/Home/blob/dev/proposed/2020/PackageVulnerability/FlagVulnerablePackages.md#display-vulnerabilities-in-visual-studio-pmui doc says, `Installed tab shows a warning icon that leads to vulnerable package, when the package has High or Critical vulnerabilities`. Reading from this spec it looks like a warning icon will be displayed irrespective of severity of the vulnerability. It is possible that I may be missing some context here.

Reply:
@kartheekp-ms The tooltip will reflect the maximum severity of the vulnerabilities on the package. The icon will be the same regardless of severity.